src/Microsoft.Azure.ServiceBus/Core/MessageReceiver.cs

```diff
@@ -67,26 +175,35 @@ internal set
             }
         }
 
+        public virtual string Path
+        {
+            get { return this.entityPath; }
```

Comment:
Similarly for others

---
In reply to: [113600980](https://github.com/Azure/azure-service-bus-dotnet/pull/146#discussion_r113600980) [](ancestors = 113600980)